contracts/src/v0.8/mocks/MockAggregator.sol

```diff
@@ -1,8 +1,11 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-contract MockAggregator {
```

Comment:
Please move all automation code to an `automation folder` this includes tests, mocks, source, everything